ci/azure-pipelines.yml

```diff
@@ -24,3 +24,16 @@ jobs:
   - bash: tools/generate_go_protobuf.py && test -z "$(git status --porcelain)"
     env:
       TEST_TMPDIR: $(Agent.TempDirectory)/tmp
+
+- job: go build
```

Comment:
not sure that spaces are allowed (https://learn.microsoft.com/en-us/azure/devops/pipelines/yaml-schema/jobs-job?view=azure-pipelines).
Can probably use `displayName` if you really want.

Reply:
Ah, ok. I wish I was able to test-run this stuff.

Reply:
Ah, I see. That's why it wasn't running.